roles/usb_lib/files/usbmount/usbmount

```diff
@@ -96,22 +96,41 @@ if [ "$1" = add ]; then
     USAGE=$(echo "$DEVINFO"  | sed 's/.*[[:blank:]]USAGE="\([^"]*\)".*/\1/g; s/[[:blank:]]*//g;')
 
     if ! echo $USAGE | egrep -q "(filesystem|disklabel)"; then
-	log info "$DEVNAME does not contain a filesystem or disklabel"
+	log info "/$DEVNAME does not contain a filesystem or disklabel"
 	exit
     fi
 
-    # Try to use specifications in /etc/fstab first.
-    if egrep -q "^[[:blank:]]*$DEVNAME" /etc/fstab; then
-	log info "executing command: mount $DEVNAME"
-	mount $DEVNAME || log err "mount by DEVNAME with $DEVNAME wasn't successful; return code $?"
+    log debug "DEVNAME /$DEVNAME"
+    BOOTFW_DEV=$(/usr/bin/findmnt -no source /boot/firmware)
+    log debug "BOOTFW_DEV $BOOTFW_DEV"
+    if [ $BOOTFW_DEV = /$DEVNAME ]; then
+	log debug "/$DEVNAME contains filesystem type $FSTYPE"
+	log debug "skipping BOOTFS_DEV $BOOTFS_DEV mounted at /boot/firmware"
```

Comment:
```suggestion
	log debug "skipping BOOTFW_DEV $BOOTFW_DEV mounted at /boot/firmware"
```

Reply:
Are you referring to tabs vs spaces here? I just matched what was used elsewhere within the file.

Reply:
Typos: BOOTFS_DEV should be BOOTFW_DEV

I hadn't notice tabs.  But feel free to make that consistent (e.g. spaces across the board?)

Reply:
Yea, I'll fix that in a bit.